libs/database/src/lib/services/user.service.v2.ts

```diff
@@ -209,4 +210,117 @@ export class UserServiceV2 {
 
     return pages as PageResponseDto[];
   }
+
+  async blockUser(user: User, targetCastcleId: string) {
+    const blockUser = await this.repositoryService
+      .findUser({
+        _id: targetCastcleId,
+      })
+      .exec();
+
+    if (!blockUser || !targetCastcleId)
```

Comment:
```suggestion
    if (!blockUser)
```

targetCastcleId ถูก validate จาก controller แล้วว่าต้อง IsString + IsNotEmpty ซึ่งมันไม่โอกาศเป็น falsy value แล้วนะ ผมว่าตรงนี้ไม่จำเป็น